src/deployment/obfuscate.md

```diff
@@ -116,6 +116,23 @@ use the following steps to make it human readable:
    For more information on the `symbolize` command,
    run `flutter symbolize -h`.
 
+## Reading an obfuscated name
+
+To understand a name created by an obfuscated app,
+use the following steps to make it human readable:
+
+1. At app build time, save the name obfuscation map
+   using `--extra-gen-snapshot-options=--save-obfuscation-map=/<your-path>`.
+   For example:
+
+   ```terminal
+   $ flutter build apk --obfuscate --split-debug-info=/<project-name>/<directory> --extra-gen-snapshot-options=--save-obfuscation-map=/<your-path>
+   ```
+
+1. Use the generated obfuscation map to recover the name.
+   The obfuscation map is a simple JSON array where
+   each obfuscated name is preceded by its original name.
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).

```suggestion
1.  To recover the name, use the generated obfuscation map.
   The obfuscation map is a small JSON array where
   each obfuscated name is preceded by its original name.
```

**question:** Is this an array of JSON objects? That would give you pairs, which would be a more basic explanation.

Reply:
Yes it is pairs, I will update that